<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edgedetectWindow</class>
 <widget class="QMainWindow" name="edgedetectWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="edgedetectWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>110</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_edgedeteclabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>エッジ認識設定 
Edge Detection Params</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveparambutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>920</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>設定保存
Save Params</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1811</width>
      <height>751</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="cameraFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1791</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mainmenubutton">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>920</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt; Main Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitbutton">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>970</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_opacity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>960</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_opacitylabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>930</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Opacity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>920</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>920</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_blurlabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>920</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider_blur">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>960</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_lowercanny">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>960</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_uppercanny">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>960</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_contrast">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>960</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_brightness">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>960</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_lowercannylabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>920</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lower Canny</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_uppercannylabel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>920</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Upper Canny</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_contrastlabel">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>920</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_brightness">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>920</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
